src/components/smart/OeAdres.vue

```diff
@@ -577,8 +581,10 @@ watch(gemeente, async (selectedGemeente, oldValue) => {
     resetFreeTextState();
 
     try {
+      isLoading.value = true;
       postinfo.value = await crabApiService.getPostinfo((selectedGemeente as IGemeente).naam);
       straten.value = sortBy(await crabApiService.getStraten((selectedGemeente as IGemeente).niscode), 'naam');
+      isLoading.value = false;
```

Comment:
Moet deze niet in `finally` block? Denk dat het anders blijft laden als postinfo of straten call faalt.